react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [isOn, setIsOn] = useState(false);
+  let modifierClass = null;
```

Comment:
I think if you do this the className will be "toggle-switch null" in the `isOn` case. You can check Dev Tools to see. The best would be to simply initialize this with an empty string.